testsuite/MDAnalysisTests/coordinates/test_trc.py

```diff
@@ -0,0 +1,189 @@
+        trc.next()
+        trc.next()
+        trc.next()
+        trc.next()
+        assert_equal(trc.ts.frame, 4,
+                     "trajectory.next() did not forward to frameindex 4")
+        trc.rewind()
+        assert_equal(trc.ts.frame, 0, "trajectory.rewind() failed "
+                                      "to rewind to first frame")
+
+        assert np.any(TRC_U.atoms.positions != 0), "The atom positions " \
+                                                   "are not populated"
+
+    def test_random_access(self, TRC_U):
+        TRC_U.trajectory[0]
+        pos0 = TRC_U.atoms[0].position
+        TRC_U.trajectory.next()
+        TRC_U.trajectory.next()
+        pos2 = TRC_U.atoms[0].position
+
+        TRC_U.trajectory[0]
+
+        assert_equal(TRC_U.atoms[0].position, pos0)
+
+        TRC_U.trajectory[2]
+
+        assert_equal(TRC_U.atoms[0].position, pos2)
+
+    def test_read_frame_reopens(self, TRC_U):
+        TRC_U.trajectory._reopen()
+        TRC_U.trajectory[4]
+        assert TRC_U.trajectory.ts.frame == 4
+
+
+class TestTRCReaderSolvatedBox:
+    @pytest.fixture(scope='class')
+    def TRC_U(self):
+        return mda.Universe(TRC_PDB_SOLV, TRC_TRAJ_SOLV)
+
+    def test_trc_n_atoms(self, TRC_U):
+        assert (TRC_U.trajectory.n_atoms) == 2797
+
+    def test_periodic(self, TRC_U):
+        assert_equal(TRC_U.trajectory.periodic, True)
+
+    def test_trc_dimensions(self, TRC_U):
+        ts = TRC_U.trajectory[1]
+        assert_allclose(
+            ts.dimensions,
+            [30.54416298, 30.54416298, 30.54416298, 90., 90., 90.]
+        )
+
+    def test_open_twice(self, TRC_U):
+        TRC_U.trajectory._reopen()
+        with pytest.raises(IOError):
+            TRC_U.trajectory.open_trajectory()
+
+
+class TestTRCReaderClusterTrajectory:
+    @pytest.fixture(scope='class')
+    def TRC_U(self):
+        with pytest.warns(UserWarning) as record_of_warnings:
+            TRC_U = mda.Universe(TRC_PDB_VAC, TRC_CLUSTER_VAC, format="TRC")
+
+        warning_strins = ["The trajectory does not contain TIMESTEP blocks!",
+                          "POSITION block is not supported!"]
+
+        warning_strins_found = 0
+        for w in record_of_warnings:
+            if (str(w.message) in warning_strins):
+                warning_strins_found += 1
+
+        assert (warning_strins_found) == 2
+        return TRC_U
+
+    def test_trc_n_atoms(self, TRC_U):
+        assert (TRC_U.trajectory.n_atoms) == 73
+
+    def test_periodic(self, TRC_U):
+        assert_equal(TRC_U.trajectory.periodic, False)
+
+    def test_trc_dimensions(self, TRC_U):
+        ts = TRC_U.trajectory[1]
+        assert_equal(ts.dimensions, None)
+
+    def test_trc_n_frames(self, TRC_U):
+        assert len(TRC_U.trajectory) == 3
+        assert (TRC_U.trajectory.n_frames) == 3
+
+    def test_trc_frame(self, TRC_U):
+        # Catch expected warnings: "POSITION block is not supported!"
+        with pytest.warns(UserWarning):
```

Comment:
also check the message with the [`match=str|regex`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.warns)
```suggestion
       with pytest.warns(UserWarning, match="POSITION block is not supported!"):
```

Reply:
Yes, I agree.
Done.